src/averageAges.js

```diff
@@ -20,6 +20,16 @@ function calculateMenAverageAge(people, century) {
   // avoid using loop and forEach
   // replace `if ()` statement with &&, || or ?:
   // without nesting
+  const menByCentury = people.filter((person) =>
+    century
+      ? Math.ceil(person.died / 100) === century && person.sex === "m"
+      : person.sex === "m"
+  );
+
+  return (
+    menByCentury.reduce((sum, man) => sum + man.died - man.born, 0) /
+    menByCentury.length
```

Comment:
move into function and reuse throughout your code

Reply:
What exactly do you want me to reuse? If only what I divide by the length of the array, then it seems to me that it will not shorten the code at all

Reply:
the separate function like `calculatePeoplesAge` can be created and reused in 2 functions